templates/home.html

```diff
@@ -34,6 +34,10 @@ <h4>All Tweets</h4>
     <h6>{{ tweet.user.first_name }}</h6>
     <p>{{ tweet.tweet_message }}</p>
     <a href="{% url 'tweets:showTweet' tweet.id %}">Show Tweet</a>
+    {% if request.user ==  tweet.user %}
```

Comment:
Use
```suggestion
    {% if request.user.id ==  tweet.user_id %}
```

saves one query

Reply:
I didn't used any query to compare. Maybe django have some inbuilt function to check.

Reply:
how do you think Django with know who is the User for Tweet

Reply:
yes there must be something, but this is why we are using framework that we don't want to write large code and use inbuilt functions. I learned about this so I used it as I know and it was working fine.